sdk/spring/spring-cloud-azure-appconfiguration-config/src/main/java/com/azure/spring/cloud/appconfiguration/config/implementation/AppConfigurationPropertySourceLocator.java

```diff
@@ -260,8 +258,8 @@ private List<AppConfigurationPropertySource> create(AppConfigurationReplicaClien
         if (store.getFeatureFlags().getEnabled()) {
             for (FeatureFlagKeyValueSelector selectedKeys : store.getFeatureFlags().getSelects()) {
                 AppConfigurationFeatureManagementPropertySource propertySource = new AppConfigurationFeatureManagementPropertySource(
-                    store.getEndpoint(), client, selectedKeys.getKeyFilter(),
-                    selectedKeys.getLabelFilter(profiles));
+                    store.getEndpoint(), client,
+                    selectedKeys.getKeyFilter(), selectedKeys.getLabelFilter(profiles));
 
                 propertySource.initProperties();
```

Comment:
Why there is no need to pass `store.trim()` for feature flags? Is it becasuse the `trim` is not available for feature flags?

Reply:
The prefix on feature flags is predetermined. i.e. it is always `.appconfig.featureflag/` as defined by our design. I believe that this is consistent across all of our libraries.